templates/addons/cilium/cilium.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: addons.cluster.x-k8s.io/v1alpha1
+kind: HelmChartProxy
+metadata:
+  name: cilium
+spec:
+  clusterSelector:
+    matchLabels:
+      cni: cilium
+  repoURL: https://helm.cilium.io/
```

Comment:
something I realized we probably want to do here is to install cilium in a NS other than `default`

Reply:
good catch, updated to kube-system